tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -451,6 +451,28 @@ def from_tensor_slices(tensors):
     # Dictionary structure is also preserved.
     Dataset.from_tensor_slices({"a": [1, 2], "b": [3, 4], "c": [5, 6]})
     # ==> [ {"a": 1, "b": 3, "c": 5}, {"a": 2, "b": 4, "c:" 6} ]
+
+
```

Comment:
please remove the extra newline to make the spacing between examples consistent

Reply:
Fixed, thanks!